wordpress-mobile/WordPress-iOS
# Remove Jetpack Signup feature flag

**Fixes** #8949 

**To test:**

Check that in a production build Jetpack sign up is enabled.

Note: @nheagy while working on this I noticed two things:

- Social signup is not enabled from the Jetpack flow (I'm fine shipping a v1 without this)
- There's no option to update the username in this flow

Is there any particular reason for that or just that it's another step we haven't implemented yet?
